Toolkit/ArcGISToolkit/TimeSlider.swift

```diff
@@ -2138,6 +2180,32 @@ public class TimeSlider: UIControl {
         return dateFormatter.string(from: date)
     }
     
+    // Calculates time step interval based on provided time extent and time step count
+    private func calculateTimeStepInterval(for timeExtent: AGSTimeExtent, timeStepCount: Int) -> AGSTimeValue? {
+        if let startTime = timeExtent.startTime, let endTime = timeExtent.endTime {
+            //
+            // Calculate time step interval
+            // based on time step count
+            if timeStepCount > 0 {
```

Comment:
Since you are dividing by `timeStepCount - 1` in the line below, should the check be for `timeStepCount > 1`?  To prevent divide-by-zero?

Or can it never be 1?

Reply:
`timeStepCount` is going to be either `0` or `timeStepCount >= 2` (guard is in the `initializeTimeSteps` function from where this function is called) so the `divide-by-zero` won't happen. Do you still think I should change the condition?

Reply:
Ok, I see where the `guard` is.  I would still change the condition, as you're never sure if sometime later it will be called somewhere else.